db-migrations/01-add-collections/01-add-collections.sql

```diff
@@ -46,7 +46,7 @@ CREATE TABLE IF NOT EXISTS collections (
     versionNumber VARCHAR (255) NOT NULL,
```

Comment:
Isn't it a bad practice to modify old migrations? Is it possible to create a new migration?

Reply:
The table declaration fails on the first migration.  I don't think an additional migration can solve it.

It is generally a smell but I don't see an alternative in this case.

Reply:
or maybe I don't well enough understand the relationship between the db-migrations and setup.mysql

Reply:
It _seems_ like setup.mysql is the equivalent of `schema.rb` in that it represents the culmination of all the migrations, and the migrations are like rails migrations, but they're applied manually: https://github.com/PLOS/content-repo/blob/087201eab2dbe7042f651085ce7bb0ca830769cd/db-migrations/README#L17 

I don't see a problem with editing this migration since the alternative is that it errors if we were to ever try to run it.

Reply:
Yeah, me neither. it sucks that we have our own migration system. whatever works, I guess :(